lib/services/secure_storage/service.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
+
+/// Service for interacting with [FlutterSecureStorage].
+class SecureStorageService {
+  /// String key associated with the password.
+  static const _passwordKey = 'password';
+
+  /// Wrapped instance.
+  final FlutterSecureStorage _secureStorage;
+
+  const SecureStorageService(this._secureStorage);
+
+  /// Reads the password.
+  Future<String?> get password => _secureStorage.read(key: _passwordKey);
```

Comment:
We should store the hash of the password such that it isn’t loaded as is in memory.

Also when we use the hash I would argue we could avoid storing it in a secure storage.

Reply:
I think we decided to store the password like this and use it to encrypt the rest of the password. If we don't store it we'll need to keep it in memory to be able to access the DB (https://docs.hivedb.dev/#/advanced/encrypted_box).

I'm open to revising this but think we should start with what was previously agreed.